src/website/app/demos/Menu/examples/menu-divider/menuDivider.js

```diff
@@ -21,11 +21,11 @@ import DemoLayout from '../../components/DemoLayout';
 
 export default {
   id: 'basic',
-  title: 'Menu with Dividers',
+  title: 'Example',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description:
-    '[MenuItems](../menu-item) and [MenuGroups](../menu-group) can optionally be separated with a MenuDivider.',
+  description: `[MenuItems](../menu-item) and [MenuGroups](../menu-group) can be separated with a MenuDivider.
+MenuDividers are used to create hierarchy by grouping related options.`,
```

Comment:
Since we have MenuGroup for grouping, maybe we should emphasize MenuDivider's ability to, well, divide and set menu options apart from others?

Reply:
updated, but I'm honestly not sure what to put here. MenuDivider and MenuGroup seem to fill the same purpose, but with different levels of impact. I'd love to revisit this after we have some more guidance on the design side.